### ISSUE TITLE
can't pair once unpaired (must reinstall KDEConnect on phone)

### ISSUE THREAD

**Issue by developer91234** [andyholmes/gnome-shell-extension-gsconnect#217]
![image](https://user-images.githubusercontent.com/43324854/46383260-29a8a380-c6b8-11e8-8dca-d726eee863c4.png)

To reproduce, switch networks or click "unpair". The connection will be severed.

Then you can't actually pair it back by clicking the button above. You click it and nothing happens. The only way to reestablish connection is to unpair and re-pair from the phone.

I'm on Fedora 28 Gnome and have a Samsung Galaxy s6.

UPDATE: the device is probably still paired but GSConnect interface shows it as "upaired"

![image](https://user-images.githubusercontent.com/43324854/46383685-4ba32580-c6ba-11e8-8041-25b477a1ef7f.png)

So when you click "pair" it doesn't do anything because the devices are in fact already paired. That could be the reason.

**Comment by andyholmes** [comment]
This can happen if you unpair an offline device. In that case the device should usually repair if you ask I to, but it's necessary otherwise you could never unpair an offline device.

**Comment by andyholmes** [comment]
After thinking about it, in `master` the pair handling was rewritten, and it's been a long time since I looked at the code for <= v12.

The newer code (at least) will recognize when a remote device thinks it's still paired, but we have already unpaired it. GSConnect will then re-send the unpair request to correct situations when the remote device was offline during unpairing and doesn't "know" it's been unpaired.

At this point `master` has been getting a bit of testing from users, and is fairly stable and close to a release candidate. If you wanted to give that a try and help with testing, I think the problem may be fixed there.
